wordpress-mobile/WordPress-Android
# Implement answer prompt local notification

Fixes #16311 

<img width="329" alt="Screen Shot 2022-04-19 at 7 59 40 PM" src="https://user-images.githubusercontent.com/14964993/164142929-64c4588c-b931-454d-9420-c835dda9a439.png">

To test:
- Open `Debug settings` and enable `blogging_prompts_notification_remote_field`
- On `My Site`, tap on `Menu`
- Tap on `Site Settings`
- Tap on `Reminders and prompts` and select a date and time for the reminder
- Change the device date to something after the selected date/time for the reminder
- A notification to answer the prompt should appear
- Tap on the notification and the editor should be opened
- If the feature flag mentioned above is disabled, the existing blogging reminder notification should be shown 

## Regression Notes
1. Potential unintended areas of impact
Blog reminder local notifications

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing both notifications: existing reminders and answer prompt reminders

3. What automated tests I added (or what prevented me from doing so)
`PromptReminderNotifierTest.kt`

PR submission checklist:
